CollAction/Views/Shared/_Footer.cshtml

```diff
@@ -43,11 +43,9 @@
         <h5>@Localizer["Follow Us"]</h5>
         <ul class="social-list">
           <li><a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=http://collaction.org/"><i class="fa fa-facebook marginright"></i></a></li>
-          <li>
-            <a target="_blank" href="https://twitter.com/intent/tweet?original_referer=http://collaction.org/&amp;url=http://collaction.org/"><i class="fa fa-twitter marginright"></i></a>
-          </li>
-          <li><a target="_blank" href="http://www.pinterest.com/pin/create/button/?url=http%3A%2F%2Fcollaction.org%2F"><i class="fa fa-pinterest marginright"></i></a></li>
-          <li><a target="_blank" href="https://plus.google.com/share?url=http://collaction.org/"><i class="fa fa-google-plus marginright"></i></a></li>
+          <li><a target="_blank" class="twitter-follow-button" href="https://twitter.com/CollAction_org"><i class="fa fa-twitter marginright"></i></a></li>
```

Comment:
Are you also fixing these links on the homepage (in the 'Spread it further' box)? I'm still getting the facebook error there. Or is that a different task?